Error: cycle detected: resources.jobs.foo refers to itself via ${resources.jobs.foo.id}


Exit code (musterr): 1
